resources/osx/Info.plist

```diff
@@ -4,7 +4,7 @@
 <dict>
 
     <key>CFBundleExecutable</key>
-    <string>EasyRPG Player</string>
+    <string>Player</string>
```

Comment:
Is this string shown anywhere when the application is installed?

The full name is "EasyRPG Player" - Also "Player" is ambigious.

Reply:
I'll see if I can get cmake and friends to emit "EasyRPG Player"

Reply:
CMake supports a plist template file "plist.in". Maybe is useful: https://stackoverflow.com/questions/42313808/customising-macos-bundles-info-plist-file-with-cmake

https://cmake.org/cmake/help/v3.18/prop_tgt/MACOSX_BUNDLE_INFO_PLIST.html